src/lib/components/SnoozePopup.js

```diff
@@ -83,3 +83,8 @@ export default class SnoozePopup extends React.Component {
     }
   }
 }
+
+SnoozePopup.propTypes = {
```

Comment:
They're not used here, but they *are* [passed](https://github.com/bwinton/SnoozeTabs/blob/470d69fe7f9bfe263dc4addd5d59da2819db6e91/src/lib/components/SnoozePopup.js#L65) & used & required by all the components this component uses.